tests/constants/requirements.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (C) 2021 Intel Corporation
+#
+# SPDX-License-Identifier: MIT
+
+
+class Requirements:
+
+    # Dummy requirement
+    REQ_1 = "Dummy requirement"
```

Comment:
I suggest it to call a dummy req, or something like this. A descriptive name.

Reply:
Corrected